src/components/staking/PageValidator.vue

```diff
@@ -8,7 +8,8 @@
     data-title="Validator"
     class="small"
   >
-    <template v-if="validator.operatorAddress" slot="managed-body">
+    <TmDataLoading v-if="$apollo.loading" />
```

Comment:
```suggestion
    <TmDataLoading v-if="$apollo.queries.validator.loading" />
```

Reply:
if i'm not mistaken, this will only show when `validator` is loading - we don't need to wait for `delegation` or `rewards` to load the page.

@colw would you agree?

Reply:
Yes. We should always show something as fast as possible.